recipes/ifitpy/meta.yaml

```diff
@@ -0,0 +1,38 @@
+
+package:
+  name: "ifitpy"
+  version: "0.0.2"
+
+source:
+  url: https://github.com/gpinpereira/Pyfit/archive/refs/tags/0.0.2.tar.gz
+  sha256: e9cadfc58f52bd1cd0a757a120a0b3115041b5e303fa7ec8afc0da9f752d6ec3
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - iminuit
+    - scipy
+
+test:
+  imports:
+    - ifitpy
+
+about:
+  home: https://github.com/gpinpereira/Pyfit
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
Change to `LICENSE.txt` so it looks for the copy already in the source distribution.